src/components/sessions/SessionTimeline.tsx

```diff
@@ -124,12 +127,24 @@ function SessionTimeline({
   handleTogglePublicEvaluation,
   handleResetQuestionBlock,
   handleActivateBlockById,
+  handleActiveBlock,
+  handleNoActiveBlock,
   subscribeToMore,
 }: Props): React.ReactElement {
   useEffect((): void => {
     subscribeToMore()
   }, [])
 
+  const [isBlockActive, setIsBlockActive] = useState(activeStep % 2 === 1)
+
+  useEffect(() => {
+    setIsBlockActive(activeStep % 2 === 1)
+  }, [activeStep])
+
+  useEffect(() => {
```

Comment:
I think you could combine everything and rewrite it like:

```javascript
useEffect(() => {
	const isBlockActive = activeStep % 2 === 1
	if (isBlockActive) {
		handleActiveBlock()
	} else {
		handleNoActiveBlock()
	}
}, [activeStep])
```

Reply:
Thanks for reviewing, totally makes sense! Implemented it as suggested...